packages/eslint-plugin-tslint/package.json

```diff
@@ -33,7 +33,7 @@
     "clean": "tsc -b tsconfig.build.json --clean",
     "postclean": "rimraf dist && rimraf coverage",
     "format": "prettier --write \"./**/*.{ts,mts,cts,tsx,js,mjs,cjs,jsx,json,md,css}\" --ignore-path ../../.prettierignore",
-    "lint": "eslint . --ignore-path='../../.eslintignore'",
+    "lint": "eslint .",
```

Comment:
If I revert this change, we see that it looks like the root-level `.eslintignore` isn't getting properly resolved to this package's paths:

```plaintext
$ yarn lint
yarn run v1.22.19
$ eslint . --ignore-path='../../.eslintignore'

/Users/josh/repos/typescript-eslint/packages/eslint-plugin-tslint/tests/fixture-project/1.ts
  1:1  error    Unexpected var, use let or const instead                                        no-var
  1:5  warning  'foo' is assigned a value but never used. Allowed unused vars must match /^_/u  @typescript-eslint/no-unused-vars

...

/Users/josh/repos/typescript-eslint/packages/eslint-plugin-tslint/tests/test-tslint-rules-directory/alwaysFailRule.js
  0:0  error  Parsing error: "parserOptions.programs" has been provided for @typescript-eslint/parser.
The file was not found in any of the provided program instance(s): packages/eslint-plugin-tslint/tests/test-tslint-rules-directory/alwaysFailRule.js

✖ 14 problems (11 errors, 3 warnings)
  6 errors and 0 warnings potentially fixable with the `--fix` option.
```

Am I missing something obvious?

Reply:
Eslint doesn't recursively look up the tree I believe?

We should really switch off of the eslint ignore file in favour of config ignores IMO so we don't need to worry about this sort of thing

https://eslint.org/docs/latest/user-guide/configuring/ignoring-code#ignorepatterns-in-config-files

Reply:
Using Nx's built in executor assists with this, and it may well be more efficient than invoking the eslint CLI in a child process like this (i.e. would help with the slowdown @JoshuaKGoldberg mentioned in the uncached case)

I will put up a commit which applies this